plugins/pgpointcloud/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ PDAL_ADD_PLUGIN(reader_libname reader pgpointcloud
         ${PDAL_LIBRARIES}
     INCLUDES
         ${LIBXML2_INCLUDE_DIR}
+        ${NLOHMANN_INCLUDE_DIR}
```

Comment:
I need to include these to the plugin, as reader class include `Connector.hpp` which require both `nlohann` and `arbiter`

Reply:
I don't understand this, but I don't have time to look right now. The class can certainly be changed to eliminate these dependencies.

I think the way things are being done isn't great in general (not these changes, as such). `header` and `query` can be of some type that knows how to read itself from a string, like other arguments. Then `setForwards` is unnecessary.

I'll look more tomorrow.